pandas/io/sas/sas.pyx

```diff
@@ -294,7 +291,7 @@ cdef class Parser:
         self.parser._current_row_in_chunk_index = self.current_row_in_chunk_index
         self.parser._current_row_in_file_index = self.current_row_in_file_index
 
-    cdef bint read_next_page(self):
+    cdef read_next_page(self):
```

Comment:
Why was the `bint` here removed?

Reply:
Because otherwise `read_next_page` doesn't propagate exceptions. Alternative could be `cdef bint read_next_page(self) except *`

Reply:
The alternatives would be preferred IMO